manifests/h/Hashicorp/Terraform/1.2.9/Hashicorp.Terraform.installer.yaml

```diff
@@ -0,0 +1,25 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Hashicorp.Terraform
+PackageVersion: 1.2.9
+InstallerType: portable
```

Comment:
InstallerType: zip

Reply:
Thanks mate - I'd wondered why  `get-command terraform` failed / assumed it was down to winget not setting the path environment variable.  Nice one :)